src/main/java/seedu/address/logic/parser/patient/SearchPatientCommandParser.java

```diff
@@ -38,6 +38,8 @@ public SearchPatientCommand parse(String args) throws ParseException {
                 new SearchPatientCommand.SearchPatientDescriptor();
 
         if (argMultimap.getValue(PREFIX_NAME).isPresent()) {
+            String[] field = args.trim().split(" ");
+            searchPatientDescriptor.setStringName(field[0].substring(2));
```

Comment:
hmm, just curious ah, why dont want just ``String stringName = argMultimap.getPreamble().trim().toLowerCase()``? so eg. if you have ``searchpatient alex n/something``, your ``stringName`` will be ``alex``.

Reply:
Also, what is the ``substring(2)`` for ah? i think for very long but i still cant figure it out

Reply:
the substring(2) is to substract away the header ("n/")

Reply:
I think getPreamble will work just fine!

Reply:
actually I think getPreamble cannot work since the command is searchpatient n/alex, so there isn't a preamble for the command for now